test/parallel/taskCompare/elliot/empty-omp-parallel-for.skipif

```diff
@@ -0,0 +1,6 @@
+# skip configs where -fopenmp or /proc/cpuinfo aren't supported
+CHPL_TARGET_COMPILER==clang
+CHPL_TARGET_COMPILER<=pgi
```

Comment:
I based this skipif off of the csu [skipif] (https://github.com/chapel-lang/chapel/blob/master/test/studies/colostate/Jacobi1D-DiamondByHand-OMP_dyn.skipif) so I think it'll be ok. 

I also don't think we'll actually run under icc right now since the test will be skipped for whitebox and cray performance testing